scripts/travisPublishDocs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Set identity
+git config --global user.email "travis-ci-hyperwallet@paypal.com"
+git config --global user.name "travis-ci-hyperwallet"
+
+# Publish docs
+mkdir ../gh-pages
+cp -r doc/_build/html/* ../gh-pages/
```

Comment:
updated the docs folder path to what it should be in swift

Reply:
Updated